Adyen/UI/List/ListItemView.swift

```diff
@@ -168,7 +167,7 @@ public final class ListItemView: UIView, AnyFormItemView {
 
         trailingTextLabel.setContentHuggingPriority(.required, for: .horizontal)
         
-        NSLayoutConstraint.activate(constraints)
+        NSLayoutConstraint.activate(viewConstraints)
```

Comment:
if we don't need those changes, can you revert them please?

Reply:
yes.